doc/md/fundamentals/expressions/operators.md

```diff
@@ -4,11 +4,131 @@ sidebar_position: 2
+| Text       | Text concatenation              | `#` |
+| Logical | Logical/Boolean operations       | `!`, `&`, |
+| Ordered | Compare values                  | `==`, `!=`, `<`, `>` |
+
+## Unary operators
+
+| Operator | Description |
+|----------|------------|
+| `-`      | Numeric negation |
+| `+`      | Numeric identity |
+| `^`      | Bitwise negation |
+
+## Relational operators
+
+Relational operators compare two values and return `true` or `false`:  
+
+| Operator | Description | Example|
+|----------|------------|-----------|
+| `==`     | Equals | `a == b` |
+| `!=`     | Not equals | `a != b`|
+| `<`      | Less than | `a < b` |
+| `>`      | Greater than | `a > b` |
+| `<=`     | Less than or equal | `a <= b` |
+| `>=`     | Greater than or equal | `a >= b`|
+
+## Numeric binary operators
+
+Binary operators combine two numbers to produce a result.
+
+| Operator | Description |  Example|
+|----------|------------|------------|
+| `+`      | Addition | `a + b` |
+| `-`      | Subtraction | `a - b` |
+| `*`      | Multiplication | `a * b` |
+| `/`      | Division (integer division) | `a / b` |
+| `%`      | Modulus (remainder) | `a % b` |
+| `**`     | Exponentiation | `a ** b` |
+
+⚠ **Note:**  Division (`/`) on integers **truncates** decimals. For floating-point division, use `Float.fromInt()`.  
+
+```motoko
+let result = Float.fromInt(10) / Float.fromInt(3);
+```
+
+## Bitwise operators
+
+Bitwise operators manipulate numbers **at the binary level**.  
+
+| Operator | Description |Example |
+|----------|------------|----------|
+| `&`      | Bitwise AND |`a & b` |
+|  `|`    | Bitwise OR | `a | b` |
+| `^`      | Bitwise XOR | `a ^ b` |
+| `<<`     | Shift left | `a << b` |
+| `>>`     | Shift right (must be proceeded by whitespace) |`a >> b` |
+| `<<>`    | Rotate left (circular shift) | `a <<> b` |
+| `<>`     | Rotate right (circular shift)| `a <> b` |
+
+## Wrapping operators
+
+Bounded integers **trap** on overflow, but **wrapping versions** allow overflow behavior:
+
+| Operator | Description | Example |
+|----------|------------|------------|
+| `+%`     | Addition with wrap-around | `a +% b` |
+| `-%`     | Subtraction with wrap-around | `a -% b` |
+| `*%`     | Multiplication with wrap-around | `a *% b` |
+| `**%`    | Exponentiation with wrap-around | `a **% b` |
+
+## Text operators
+
+| Operator | Description | Example |
+|----------|------------|------------|
+| `#`      | Concatenates two `Text` values | `a # b` |
+
+## Assignment operators
+
+Assignment operators modify variables in place. Only mutable variables `var` can be reassigned.  
+
+| Operator | Description |Examples|
+|----------|------------|---------|
+| `:=`     | Assign a value | `a := b` |
+| `+=`     | Add and assign | `a += b` |
+| `-=`     | Subtract and assign | `a -= b` |
+| `*=`     | Multiply and assign | `a *= b` |
+| `/=`     | Divide and assign | `a /= b` |
+| `#=`     | Concatenate and assign (for `Text`) | `a #= b` |
+
```

Comment:
I think, but you'd jave to check that |= and &= and ^= are also supported. See the reference manual (or better, try it out)